tests/e2e/snackbar.wdio-spec.js

```diff
@@ -1,56 +0,0 @@
-const loginPage = require('../page-objects/login/login.wdio.page');
```

Comment:
I have to say this is extreme :))) 
Another option would be to disable the test with `xit`. I think that there are some floating that this test requires, but aren't used in other places, like all the `snackbar*` exports from the common elements page.

Reply:
I thought about xit, but unless it's just a short term suspension I prefer it just living on in git history where it doesn't need to be maintained or refactored.

I've removed the orphaned code.